src/bootstrap/test.rs

```diff
@@ -1299,6 +1299,13 @@ host_test!(RunMakeFullDeps {
 
 default_test!(Assembly { path: "tests/assembly", mode: "assembly", suite: "assembly" });
 
+host_test!(RunCoverage { path: "tests/run-coverage", mode: "run-coverage", suite: "run-coverage" });
+host_test!(RunCoverageRustdoc {
+    path: "tests/run-coverage-rustdoc",
+    mode: "run-coverage",
+    suite: "run-coverage-rustdoc"
+});
+
```

Comment:
I'm not fully clear on the differences between `default_test!` and `host_test!`, but since the tests are skipped on cross-compile targets anyway I assume it doesn't matter much right now.

(It should be possible to run coverage tests on cross-compile targets, but the cross-runner would need to be taught how to send back the resulting `.profraw` files to be processed by the host.)

Reply:
I also don't have strong feelings on the name of the extra suite; I was wavering between `run-coverage-doctest`, `run-coverage-doctests`, and `run-coverage-rustdoc`.

(If there are ever other run-coverage tests with extra tool dependencies, it would probably be fine to group them all into one extra suite with a more generic name.)